src/server/services/wfs3/qgswfs3handlers.cpp

```diff
@@ -1085,6 +1085,19 @@ json QgsWfs3CollectionsItemsHandler::schema( const QgsServerApiContext &context
         }
       }
     };
+
+#ifdef HAVE_SERVER_PYTHON_PLUGINS
```

Comment:
These operations should also be removed if they are not checked in layer server project settings with no plugin involved, right?

![image](https://github.com/user-attachments/assets/beb34863-2ab1-4378-9887-72fd4668a747)

Reply:
Thinking further about, looks like the proper fix would be not to add operations which are not allowed by project settings in first place (which is currently the case).

So likely out of scope for this PR.

Reply:
Yes: that should be addressed in a separate PR. There should also be a check for layer read only and for provider's editing capabilities. It is probably a good idea to create an utility function for that.